Packs/PassiveTotal/Scripts/script-PTEnrich.yml

```diff
@@ -78,4 +78,5 @@ dependson:
   - pt-ssl-cert
 timeout: 0s
 tests:
-  - No test
+- No test
+fromversion: 4.1.0
```

Comment:
Why?

Reply:
this change is caused using command `demisto-sdk format -i Packs/PassiveTotal`